### DISCUSSION TITLE
Homebrew does not install on OS X 10.8

### DISCUSSION THREAD

**Discussion by Piipperi** [Homebrew/discussions#2296] [Getting started]
I get an error about Homebrew being only supported on Intel and ARM processors, when I have an Intel Core 2 Duo... I have brew already on the Mac too but it doesn't want to update for some reason.

**Comment by carlocab** [comment]
Homebrew won't run on anything older than macOS 10.10, unfortunately. If you're unable to upgrade your Mac, try [MacPorts](https://www.macports.org).

**Reply by Piipperi** [reply]
Ye already using MacPorts on that but since I'm more familiar with Homebrew it would be nice to just use that. But guess I'll use MacPorts for now then
